docs/usage/serve.rst.inc

```diff
@@ -8,7 +8,9 @@ borg serve
 
 optional arguments
     ``--restrict-to-path PATH``
-        | restrict repository access to PATH
+        | restrict repository access to PATH.  Can be
+        | specified multiple times to allow the client
+        | access to several directories.
```

Comment:
you need to do this change in archiver.py - the *.inc files are created automatically from there.